connectors/src/connectors/slack/bot.ts

```diff
@@ -246,11 +246,20 @@ async function botAnswerMessage(
     throw new Error("DUST_FRONT_API environment variable is not defined");
   }
 
+  let requestedGroups: string[] | undefined = undefined;
+
+  if (slackUserInfo.is_bot) {
+    const botName = slackUserInfo.real_name || slackUserInfo.display_name;
+    if (typeof botName === "string") {
```

Comment:
This feels weird 🤔. What's the type of `botName`?

Reply:
Indeed, that weird, I did not want to challenge the type `SlackUserInfo`, and we do that when we check in the database if the bot has been whitelisted, but I changed the type `SlackUserInfo` to make `real_name` non nullable, and we'll use that field to check on the bot name from now on. Pushing a commit soon.